.github/scripts/release.py

```diff
@@ -1,7 +1,7 @@
 
 import argparse, copy, hashlib, json, re, requests, os, shutil
 
-version = '0.2.0'
```

Comment:
This release.py needs review together with the diff that we have now in master vs develop. By the way, we should merge the master into the develop after the last PR.

Reply:
Sure. We could review it together whenever you have time😄

Reply:
Yep, next week we can do it!